<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateItemDialog</class>
 <widget class="QDialog" name="updateItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Item</string>
  </property>
  <widget class="QLabel" name="lblItemName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string>Name of Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuantity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnconfirmEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadImage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load item image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
